modules/sdk-coin-doge/src/doge.ts

```diff
@@ -10,10 +10,49 @@ import {
   CrossChainRecoverySigned,
   CrossChainRecoveryUnsigned,
   RecoverFromWrongChainOptions,
+  Unspent,
+  TransactionInfo,
+  TransactionPrebuild,
 } from '@bitgo/abstract-utxo';
 import { BaseCoin, BitGoBase, HalfSignedUtxoTransaction, SignedTransaction } from '@bitgo/sdk-core';
 import * as utxolib from '@bitgo/utxo-lib';
 
+type UnspentJSON = Unspent<bigint> & { value: number; valueString: string };
+type TransactionInfoJSON = TransactionInfo<bigint> & { unspents: UnspentJSON[] };
```

Comment:
```suggestion
type TransactionInfoJSON = TransactionInfo<number> & { unspents: UnspentJSON[] };
```

Reply:
it doesn't make a big difference because the parameter is masked by `&` anyway, but `number` is less confusing here